apps/core/src/utils/nftMediaUtils.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { VisualAssetType } from '@iota/apps-ui-kit';
+import { transformURL } from './extractMediaFileType';
+import { NFTMediaHeadersResult } from '../hooks';
+
+const MAX_VIDEO_SIZE_MB = 50;
+const MAX_VIDEO_SIZE_BYTES = MAX_VIDEO_SIZE_MB * 1024 * 1024;
+
+export function isVideoInSizeLimit(contentLength: string | null | undefined): boolean {
+    if (contentLength) {
+        const sizeInBytes = parseInt(contentLength, 10);
+        return sizeInBytes <= MAX_VIDEO_SIZE_BYTES;
+    }
+    return false;
+}
+
+const SUPPORTED_VIDEO_MIMETYPES = ['video/mp4'];
+const SUPPORTED_IMAGE_MIMETYPES = [
+    'image/jpeg',
+    'image/png',
+    'image/gif',
+    'image/bmp',
+    'image/webp',
+    'image/x-icon',
+    'image/tiff',
```

Comment:
could you also add support for `'image'`? i have an NFT using that type, and it's not showing up in the explorer and the dashboard :see_no_evil: :pray: 

![image](https://github.com/user-attachments/assets/d2d86674-176d-4f83-9bdf-2905a5b3da98)

![image](https://github.com/user-attachments/assets/6c9fe7f3-0412-4ee4-b4d5-4a98b2a4f7cf)

Reply:
we decided that if the request to get the headers is blocked by cors and the extension is not visible in the url the image shouldn't be displayed